src/app/components/modals/modal-delete/modal-delete.component.html

```diff
@@ -0,0 +1,55 @@
+<!--
+
+  Digital Excellence Copyright (C) 2020 Brend Smits
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation version 3 of the License.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty
+   of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+   See the GNU Lesser General Public License for more details.
+
+   You can find a copy of the GNU Lesser General Public License
+   along with this program, in the LICENSE.md file in the root project directory.
+   If not, see https://www.gnu.org/licenses/lgpl-3.0.txt
+
+ -->
+
+<div class="modal-delete-container">
+  <button type="button" class="close pull-right" aria-label="Close" (click)="bsModalRef.hide()">
+    <span aria-hidden="true">&times;</span>
+  </button>
+  <div class="modal-body">
+    <h2>Delete highlight</h2>
+    <div class="row">
+      <div class="col-12">
+        <div class="highlight-form__checkbox">
+          <form [formGroup]="deleteHighlightForm" class="highlight-form">
+            <div *ngFor="let highlight of highlights">
+              <div *ngIf="highlight.isNeverEnding; else displayDateTimes">
+                <div class="highlight-wrapper">
+                  <input type="checkbox" id="neverEndingCheckbox" [formControlName]="highlight.id" class=""/>
+                  <label for="neverEndingCheckbox"><b>Never Ending</b></label>
+                </div>
+              </div>
+              <ng-template #displayDateTimes>
+                <div class="highlight-wrapper">
+                  <input type="checkbox" id="checkbox" [formControlName]="highlight.id" class=""/>
+                  <label for="checkbox"><b>Start date: </b>{{highlight.startDate}} <span class="date-spacing"></span><b>End date: </b>{{highlight.endDate}}</label>

```

Comment:
```suggestion
                  <label for="checkbox"><strong>Start date: </strong>{{highlight.startDate}} <span class="date-spacing"></span><strong>End date: </strong>{{highlight.endDate}}</label>

```
`<strong>` should be used instead of `<b>`, see: https://gyazo.com/10ba9e9397f4c94f9a59d02730a7bdf4